src/features/user/TreeMapper/components/PlantLocationPage.tsx

```diff
@@ -122,30 +122,51 @@ export function LocationDetails({
             </div> */}
         {location.measurements && (
           <>
-            <div className={styles.singleDetail}>
-              <p className={styles.title}>{t('height')}</p>
-              <div className={styles.value}>
-                {location.measurements?.height}
-                {t('m')}
+            <div className={styles.measurements}>
+              <div className={styles.singleDetail}>
+                <p className={styles.title}>{t('measurements')}</p>
               </div>
             </div>
-            <div className={styles.singleDetail}>
-              <p className={styles.title}>{t('width')}</p>
-              <div className={styles.value}>
-                {location.measurements?.width}
-                {t('cm')}
+            <div className={styles.measurements}>
+              <div className={styles.singleDetail}>
+                <p className={styles.title}>{t('date')}</p>
+                {location.history?.map((h, index) => (
+                  <div className={styles.value} key={index}>
+                    {formatDate(h?.created)}
```

Comment:
Works for me, but I would give the creation date a little more horizontal space to avoid breaking the date into two lines:

<img width="398" alt="Bildschirm­foto 2022-12-18 um 13 19 39" src="https://user-images.githubusercontent.com/1532418/208298052-91ea958b-3899-422d-9a4a-3d0fe7330384.png">